ui/helpers/utils/conversions.util.js

```diff
@@ -135,12 +135,13 @@ export function decGWEIToHexWEI(decGWEI) {
   });
 }
 
-export function hexWEIToDecGWEI(decGWEI) {
+export function hexWEIToDecGWEI(decGWEI, options = {}) {
```

Comment:
[nit] What about using destructuring here, e.g. `{ numberOfDecimals } = {}`? This should allow you to avoid the null check (`?.`).